lib/requeue.rb

```diff
@@ -2,38 +2,40 @@
 
 module Requeue
   class Queue
-    def initialize(redis: Redis.current, prefix: 'queue', unique:true)
-      @prefix = prefix 
-      @unique = unique 
+    attr_reader :redis, :prefix, :unique
+
+    def initialize(redis: Redis.current, prefix: 'queue', unique: true)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.